src/roles/common/tasks/find-file.yml

```diff
@@ -2,7 +2,7 @@
 - name: "Search for {{ pattern }} File"
   find:
     file_type: any
-    path: "{{ dir_name | default(nuage_unzipped_files_dir) }}"
+    path: "{{ dir_name | default(nuage_unzipped_files_dir) | relative_path_to_absolute_path }}"
```

Comment:
Let's use the filter when we set the paths in group_vars instead of here. I think you've already used it when setting nuage_unzipped_files_dir, so this is unnecessary.